utils/src/utils/document_plan.clj

```diff
@@ -1,60 +1,64 @@
+                        (json/read-value (:documentPlan dp) read-mapper))]
+    (-> fixed-dp
+        (document-plan->semantic-graph)
+        (pprint))))
 
-(defn ouput-document-plan [dir dp])
+(defn doc->dir-name [{kind :kind}]
+  (condp = kind
+    "RGL" "dlg" "Document" "dp" "AMR" "amr"))
 
 (defn ->file [output-dir {:keys [id] :as document-plan}]
-  (let [fpath (format "%s/%s.json" output-dir id)]
-    (log/debugf "Writing to: %s" fpath)
-    (spit
-     fpath
-     (json/write-value-as-string document-plan))))
+  (let [dir (str output-dir "/" (doc->dir-name document-plan))]
+    (.mkdirs (io/file dir))
+    (log/infof "Writing: %s/%s.json" dir id)
+    (spit (format "%s/%s.json" dir id)
+          (json/write-value-as-string document-plan))))
+
+(defn run-query [url q]
+  @(http/post url {:headers {"Content-Type" "application/json"}
+                   :body    (->> q
+                                 :graphql
+                                 (json/write-value-as-string))}))
 
 (defn export-document-plan [name]
-  (let [{:keys [graphql-url]} config
-        {:keys [status body error]} @(http/post graphql-url {:headers {"Content-Type" "application/json"}
-                                                             :body (->> (queries/export-document-plan-query {:name name})
-                                                                        (log/spyf "Query Content: %s")
-                                                                        :graphql
-                                                                        (json/write-value-as-string))})]
-    (-> (json/read-value body read-mapper) :data :documentPlan (json/write-value-as-string) (pprint))))
+  (-> (run-query (:graphql-url config)
+                 (queries/export-document-plan-query {:name name}))
+      :body
+      (json/read-value read-mapper)
+      :data :documentPlan
+      (json/write-value-as-string)
+      (pprint)))
 
 (defn export-all-document-plans
   ([] (export-all-document-plans "../api/resources/document-plans"))
   ([output-dir]
-   (let [{:keys [graphql-url]} config
-         {:keys [status body error]} @(http/post graphql-url {:headers {"Content-Type" "application/json"}
-                                                              :body (->> (queries/export-document-plans-query {})
-                                                                         (log/spyf "Query Content: %s")
-                                                                         :graphql
-                                                                         (json/write-value-as-string))})]
+   (let [{:keys [body error]} (run-query (:graphql-url config)
+                                         (queries/export-document-plans-query {}))]
      (if error
-       (log/errorf "Failed, exception is: %s" error)
+       (log/errorf "Failed with the error: %s" error)
        (doseq [dp (-> (json/read-value body read-mapper) :data :documentPlans :items)]
```

Comment:
again, `(update :documentPlan #(json/read-value % read-mapper))` is needed to match previous export format